app/lib/Config.scala

```diff
@@ -4,27 +4,34 @@ import java.io.{File, FileNotFoundException}
 import java.net.URI
 import java.text.SimpleDateFormat
 import java.util.Calendar
+import software.amazon.awssdk.services.s3.S3Client
+import software.amazon.awssdk.auth.credentials
+import software.amazon.awssdk.auth.credentials.AwsCredentialsProviderChain
+import software.amazon.awssdk.regions.{Region => AWSRegion}
 
-import com.amazonaws.auth.profile.ProfileCredentialsProvider
-import com.amazonaws.auth.{AWSCredentialsProviderChain, InstanceProfileCredentialsProvider}
-import com.amazonaws.services.s3.{AmazonS3, AmazonS3ClientBuilder}
 import scala.io.Source
 import scala.util.Try
 
 trait Config {
   // config properties required for each new app using the S3-Uploader
   val bucketName: String
   val region: String
-  val s3Client: AmazonS3
+  val s3Client: S3Client
   def key (fileName: String): String
   val maybePrettyBaseUrl: Option[String]
 
   val properties: Map[String, String] = Try(Properties.fromPath("/etc/gu/s3-uploader.properties")).getOrElse(Map.empty)
 
-  val awsCredentials = new AWSCredentialsProviderChain(
-    new ProfileCredentialsProvider("media-service"),
-    InstanceProfileCredentialsProvider.getInstance()
-  )
+  val ProfileName = "media-service"
+
+  lazy val credentialsProvider: AwsCredentialsProviderChain =
+    credentials.AwsCredentialsProviderChain
+      .builder()
+      .credentialsProviders(
+        credentials.ProfileCredentialsProvider.create(ProfileName),
```

Comment:
To match the suggestion above:


```suggestion
        credentials.ProfileCredentialsProvider.create(profileName),
```